docs/ci.md

```diff
@@ -4,17 +4,17 @@ Currently the `setup-python` GitHub Action [does not
 support](https://github.com/actions/setup-python/issues/771) installing a
 free-threaded build. For now, the easiest way to get a free-threaded Python
 build on a CI runner is with the `deadsnakes` Ubuntu PPA and the
-`deadsnakes-action` GitHub Action:
+[`deadsnakes-action` GitHub Action](https://github.com/deadsnakes/action):
 
 ```yaml
 jobs:
   free-threaded:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@...
-    - uses: deadsnakes/action@...
-      with:
-          python-version: '3.13.0b2'
+      - uses: actions/checkout@...
+      - uses: deadsnakes/action@...
+        with:
```

Comment:
For what it's worth, the NumPy github actions config doesn't have these tabs. I have no idea if adding them is a no-op. Hopefully it is but worth double-checking.

Reply:
This is a no-op, yes, and more idiomatic as far as I understand. https://stackoverflow.com/a/71968274 for more context.